src/less/variables/ds6/button-variables.less

```diff
@@ -3,7 +3,7 @@
 
 @button-icon-only-border-radius: @border-radius-small;
 
-@button-min-width: 128px;
+@button-min-width: 280px;
```

Comment:
Why 280px?

This is what I see in design:

![Screen Shot 2021-06-16 at 10 10 06 AM](https://user-images.githubusercontent.com/38065/122263254-14392f00-ce8b-11eb-85fb-a12a447d0036.png)